.changes/unreleased/Features-20230620-224944.yaml

```
@@ -0,0 +1,6 @@
+kind: Features
+body: Add support for dynamic tables
```

```suggestion
body: Add support for dynamic tables, the Snowflake implementation of materialized views
```
or something like that 🤷‍♀️

---

Snowflake also has materialized views. There is a desire to not use "materialized view" in any of the user facing components in order to reduce confusion.

---

Oh? Interesting. Makes sense to leave as is